src/components/BudgetExpiryAlertAndModal/data/expiryThresholds.js

```diff
@@ -3,76 +3,168 @@ import parse from 'html-react-parser';
 import { PLAN_EXPIRY_VARIANTS } from './constants';
 
 const expiryThresholds = {
-  120: ({ date }) => ({
+  120: ({ intl, date }) => ({
     notificationTemplate: {
-      title: 'Your Learner Credit plan expires soon',
+      title: intl.formatMessage({
+        id: 'admin.portal.learner.credit.expiry.notification.title',
```

Comment:
We had agreed on a certain naming convention for message IDs moving forward.
It would be better to use that naming pattern for IDs.
https://twou.slack.com/archives/C05QRSSP36V/p1716969849579479